src/viewer.js

```diff
@@ -521,7 +521,7 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         this.close();
 
         if (!tileSources) {
-            return;
+            return this;
```

Comment:
Great that you caught all these!

Reply:
ESLint caught them, but I turned the errors on! ;)  Yes, it was nice to see errors that needed fixing!